app/views/welcome/index.html.erb

```diff
@@ -1,4 +1,14 @@
 <h1>ShiftWork App</h1>
+<p>
```

Comment:
@saradotlog I added appropriate 'Sign In' and 'Log In' links to the Welcome page